src/main/scala/mesosphere/marathon/core/election/ElectionModule.scala

```diff
@@ -0,0 +1,64 @@
+package mesosphere.marathon.core.election
+
+import akka.actor.ActorSystem
+import akka.event.EventStream
+import com.codahale.metrics.MetricRegistry
+import mesosphere.chaos.http.HttpConf
+import mesosphere.marathon.MarathonConf
+import mesosphere.marathon.Features
+import mesosphere.marathon.core.base.ShutdownHooks
+import mesosphere.marathon.core.election.impl.{
+  ExponentialBackoff,
+  CuratorElectionService,
+  TwitterCommonsElectionService,
+  PseudoElectionService
+}
+import mesosphere.marathon.metrics.Metrics
+
+class ElectionModule(
+    config: MarathonConf,
+    system: ActorSystem,
+    eventStream: EventStream,
+    http: HttpConf,
+    metrics: Metrics = new Metrics(new MetricRegistry),
+    hostPort: String,
+    shutdownHooks: ShutdownHooks) {
+  private lazy val backoff = new ExponentialBackoff(name = "offerLeadership")
+  lazy val service: ElectionService = if (config.highlyAvailable()) {
+    if (config.isFeatureSet(Features.TWITTER_COMMONS)) {
```

Comment:
I'd add a new `--election_service <curator/twitter_commons>` flag and make it hidden.

We don't want to have to support both implementations in the future.